CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Discljord follows semantic versioning.
 - Fix missing implementation for `add-channel-pinned-message!`, delegate to new endpoints `pin-`/`unpin-message`
 - Fix typo in `modify-guild-role!` endpoint name (`modifiy` -> `modify`)
 - Fix typo in `start-thread-without-message!` which prevented it from working
+- Fix issue #108 - dependencies have slowly fallen out of date
```

Comment:
Does Github automatically detect this and make a ref? If not could you make this a markdown link?

Reply:
It appears it doesn't, which is annoying given that it does elsewhere (e.g. in PRs).  No fixed.